### ISSUE TITLE
Reproduce the deeplabv3 baseline in paper

### ISSUE THREAD

**Issue by wugh** [Gaoyiminggithub/Graphonomy#10]
Hi, 
I try to reproduce your deeplabv3+ baseline on CIHP data, But I don't know your paper result about CIHP dataset is based on Graphonomy code or [Google Deeplabv3 code](https://github.com/tensorflow/models/tree/master/research/deeplab). 
I use the google official code to train model on CIHP data, but only get MIOU about **50%**, much lower than your paper result **57.13**.

My implement detail is following:
1. Use the [coco pretrain](http://download.tensorflow.org/models/xception_65_coco_pretrained_2018_10_02.tar.gz) model provide by deeplabv3 
2. Then I train the model similar to your paper setting
```
python deeplab/train.py \
    --training_number_of_steps=180000 \
    --train_split="train" \
    --model_variant="xception_65" \
    --atrous_rates=6 \
    --atrous_rates=12 \
    --atrous_rates=18 \
    --output_stride=16 \
    --decoder_output_stride=4 \
    --train_crop_size="513,513" \
    --train_batch_size=16 \
    --base_learning_rate=0.007 \
    --fine_tune_batch_norm=True \
    --initialize_last_layer=False \
    --dataset="cihp" \
    --tf_initial_checkpoint=deeplab/pretrain/xception_65_coco_pretrained/x65-b2u1s2p-d48-2-3x256-sc-cr300k_init.ckpt \
    --train_logdir=datasets/cihp/train/xception_65_coco_pretrained_cihp_batch16 \
    --dataset_dir=deeplab/datasets/cihp/tfrecord/
```
3. The google deeplab already include the `random crop`, `scale` and `left right flip` in their code
4. But finally I only get MIOU 50%

So is there any big issue of my training precess. 
I find my model can not distinguish left and right hand and leg, is there any problem about flip?

Result from google code
![result](https://user-images.githubusercontent.com/1518998/59918920-ac48dd00-9458-11e9-951e-30736aedc695.png)

Result use your code (using `data/pretrained_model/inference.pth` model)
![xxx](https://user-images.githubusercontent.com/1518998/59918932-b074fa80-9458-11e9-8569-dd5c3540fc22.png)

My model result is not very smooth and can not class left and right.
I'm new to semantic segment, Could you please give me some advice on this issue.
Thanks 
Gordon.

**Comment by Gaoyiminggithub** [comment]
Hi, 
> My model result is not very smooth and can not class left and right.
A: you can not just use the 'left right flip' in google code, because the left/right label can not direct flip. (i.e., when you flip an image and its label, the left/right label is wrong. For example, in your image, when you flip the label and image, the label of right hand in image is left hand.) Maybe it is one of the reason that your results is bad.

**Comment by wugh** [comment]
> Hi,
> 
> > My model result is not very smooth and can not class left and right.
> > A: you can not just use the 'left right flip' in google code, because the left/right label can not direct flip. (i.e., when you flip an image and its label, the left/right label is wrong. For example, in your image, when you flip the label and image, the label of right hand in image is left hand.) Maybe it is one of the reason that your results is bad.

Thanks for your reply. But why the reply content is empty.

**Comment by Gaoyiminggithub** [comment]
Hi, 
Q: My model result is not very smooth and can not class left and right.
A: you can not just use the 'left right flip' in google code, because the left/right label can not direct flip. (i.e., when you flip an image and its label, the left/right label is wrong. For example, in your image, when you flip the label and image, the label of right hand in image is left hand.) Maybe it is one of the reason that your results is bad.

**Comment by wugh** [comment]
Thank you for your advise, I will try again.
